onnxruntime/core/optimizer/identical_children_consolidation.cc

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+#include "core/graph/graph_utils.h"
+#include "identical_children_consolidation.h"
+namespace onnxruntime {
```

Comment:
add extra space lines

Reply:
You can use clang-format to format automatically with config: [https://github.com/microsoft/onnxruntime/blob/main/.clang-format](https://github.com/microsoft/onnxruntime/blob/main/.clang-format)

Reply:
checked